### PR TITLE
[Routing] Remove Doctrine annotations support

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| Branch?       | 7.0
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | Follows #49358
| License       | MIT
| Doc PR        | symfony/symfony-docs#18589

### CODE REVIEW

File: src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/LocalizedPrefixMissingLocaleActionController.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Symfony\Component\Routing\Tests\Fixtures\AttributeFixtures;
+
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route(path: ['nl' => '/nl'])]
+class LocalizedPrefixMissingLocaleActionController
```
Comment (derrabus) [inline src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/LocalizedPrefixMissingLocaleActionController.php]: Looks like we need to backport these two fixtures and fix the tests that use them. I'll have a look.

Comment (derrabus) [reply src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/LocalizedPrefixMissingLocaleActionController.php]: see #51084

Comment (nicolas-grekas) [reply src/Symfony/Component/Routing/Tests/Fixtures/AttributeFixtures/LocalizedPrefixMissingLocaleActionController.php]: merged up, rebase needed here now

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @derrabus.

**Pull request merged**